server/src/main/java/io/druid/server/lookup/cache/LookupCoordinatorManager.java

```diff
@@ -333,115 +332,121 @@ public LookupExtractorFactoryMapContainer getLookup(final String tier, final Str
     return tierLookups.get(lookupName);
   }
 
+  public boolean isStarted()
+  {
+    return lifecycleLock.isStarted();
+  }
+
+  @VisibleForTesting
+  boolean awaitStarted(long waitTimeMs)
+  {
+    return lifecycleLock.awaitStarted(waitTimeMs, TimeUnit.MILLISECONDS);
+  }
+
   // start() and stop() are synchronized so that they never run in parallel in case of ZK acting funny or druid bug and
   // coordinator becomes leader and drops leadership in quick succession.
   public void start()
   {
-    synchronized (lifecycleLock) {
-      if (!lifecycleLock.canStart()) {
-        throw new ISE("LookupCoordinatorManager can't start.");
-      }
+    if (!lifecycleLock.canStart()) {
```

Comment:
Can you update the comment above to show that the synchronization happens on the lock inside DruidCoordinator?